A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/systems.cfg

```diff
@@ -5,37 +5,45 @@ minor = 0
-lightdef.28 = Type:13#Index:0#LocalPosition:39,-1.7,5.4#LocalRotation:-6,0,180#EffectFile:LIGHT_ASOBO_ScreenBlue#PotentiometerIndex:1
-lightdef.29 = Type:13#Index:0#LocalPosition:39,1.7,5.4#LocalRotation:-6,0,180#EffectFile:LIGHT_ASOBO_ScreenBlue#PotentiometerIndex:1
-lightdef.30 = Type:13#Index:0#LocalPosition:36.5,0,6.5#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiant#PotentiometerIndex:1
-lightdef.31 = Type:12#Index:1#LocalPosition:38.4,-1.8,5.9#LocalRotation:0,0,0#EffectFile:LIGHT_ASOBO_Glareshield#PotentiometerIndex:81
-lightdef.32 = Type:12#Index:2#LocalPosition:38.4,1.8,5.9#LocalRotation:0,0,0#EffectFile:LIGHT_ASOBO_Glareshield#PotentiometerIndex:82
+lightdef.21 = Type:10#Index:0#LocalPosition:34.8,-1.7,8.7#LocalRotation:100,0,0#EffectFile:LIGHT_A32NX_CockpitSpotLarge#PotentiometerIndex:7
+lightdef.22 = Type:10#Index:0#LocalPosition:34.8,1.7,8.7#LocalRotation:100,0,0#EffectFile:LIGHT_A32NX_CockpitSpotLarge#PotentiometerIndex:7
+lightdef.23 = Type:11#Index:0#LocalPosition:35.2,0,8.5#LocalRotation:80,0,0#EffectFile:LIGHT_A32NX_A320_Pedestal#PotentiometerIndex:76
+lightdef.24 = Type:12#Index:3#LocalPosition:38.97,-2.550,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
+lightdef.25 = Type:12#Index:3#LocalPosition:38.97,-0.6,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
+lightdef.26 = Type:12#Index:3#LocalPosition:38.97,0.7,6.2#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
+lightdef.27 = Type:12#Index:3#LocalPosition:38.97,2.55,5.87#LocalRotation:-10,0,0#EffectFile:LIGHT_A32NX_Glareshield#PotentiometerIndex:83
+lightdef.28 = Type:13#Index:0#LocalPosition:39,-1.7,5.4#LocalRotation:-6,0,170#EffectFile:LIGHT_A32NX_ScreenBlue#PotentiometerIndex:88
+lightdef.29 = Type:13#Index:0#LocalPosition:39,1.7,5.4#LocalRotation:-6,0,190#EffectFile:LIGHT_A32NX_ScreenBlue#PotentiometerIndex:90
+lightdef.30 = Type:13#Index:0#LocalPosition:36.5,0,6.5#LocalRotation:0,0,0#EffectFile:LIGHT_A32NX_CockpitMinimalAmbiant#PotentiometerIndex:83
+lightdef.31 = Type:12#Index:1#LocalPosition:38.4,-1.9,5.9#LocalRotation:100,0,0#EffectFile:LIGHT_A32NX_CockpitSpot#PotentiometerIndex:81
+lightdef.32 = Type:12#Index:2#LocalPosition:38.4,1.9,5.9#LocalRotation:100,0,0#EffectFile:LIGHT_A32NX_CockpitSpot#PotentiometerIndex:82
+lightdef.33 = Type:13#Index:0#LocalPosition:39,-0.7,4.7#LocalRotation:5,0,170#EffectFile:LIGHT_A32NX_ScreenBlue#PotentiometerIndex:89
+lightdef.34 = Type:13#Index:0#LocalPosition:39,0.4,4.7#LocalRotation:5,0,190#EffectFile:LIGHT_A32NX_ScreenBlue#PotentiometerIndex:91
+lightdef.35 = Type:13#Index:0#LocalPosition:35,0.5,5.4#LocalRotation:5,0,170#EffectFile:LIGHT_A32NX_ScreenBlue#PotentiometerIndex:92
+lightdef.36 = Type:13#Index:0#LocalPosition:40,0.5,5.4#LocalRotation:10,0,170#EffectFile:LIGHT_A32NX_ScreenBlue#PotentiometerIndex:93
```

Comment:
maybe these 4 should be `1` as well?

Reply:
Hi Aden, The screen blue light is controlled by the individual display brightness knobs. Therefore all those lights are tied to the potentiometer index of each display brightness knobs. For e.g. when you turn off  all the displays, there should not be any blue light in the cockpit. And when you turn on the displays and start rotating the display brightness knobs, the screen blue effect will also increase accordingly.  BTW i have also tried changing them to 1 and here is how it is looking with and without changes.

**Without Changes :**

1. All Displays OFF (No blue effect)

![Desktop Screenshot 2020 09 22 - 06 20 35 11](https://user-images.githubusercontent.com/53265961/93835407-c0ebd480-fc9c-11ea-8c30-2ba5f6341ddc.png)

2. Turning only the Left PFD to ON

![Desktop Screenshot 2020 09 22 - 06 29 34 80](https://user-images.githubusercontent.com/53265961/93835568-4b343880-fc9d-11ea-934f-d7e73bb68ae3.png)

3. Turning Left ND to ON

![Desktop Screenshot 2020 09 22 - 06 29 51 62](https://user-images.githubusercontent.com/53265961/93835579-55563700-fc9d-11ea-9f11-c5fa224f42e4.png)

4. Turning Both EICAS & ECAM to ON

![Desktop Screenshot 2020 09 22 - 06 30 15 78](https://user-images.githubusercontent.com/53265961/93835620-728b0580-fc9d-11ea-8004-757f7e039582.png)

5. Turning Right ND to ON

![Desktop Screenshot 2020 09 22 - 06 30 29 12](https://user-images.githubusercontent.com/53265961/93835666-9bab9600-fc9d-11ea-879b-c2292e4790e7.png)

6. Turning Right PFD to ON (All Displays are ON)

![Desktop Screenshot 2020 09 22 - 06 30 51 57](https://user-images.githubusercontent.com/53265961/93835692-aebe6600-fc9d-11ea-8f63-a1cfff205e6e.png)

**With Changes (i.e. By changing the potentiometer index value to 1 for all of them)**

1. The blue screen lights are by default ON, even though the displays are OFF.

![Desktop Screenshot 2020 09 22 - 06 15 56 62](https://user-images.githubusercontent.com/53265961/93835751-e200f500-fc9d-11ea-82c8-6268b49341e6.png)

Please let me know.
Thanks
-Avinash

Reply:
👍